lib/manager/terragrunt-version/index.ts

```diff
@@ -0,0 +1,9 @@
+import * as hashicorpVersioning from '../../versioning/hashicorp';
+
+export { extractPackageFile } from './extract';
+
+export const defaultConfig = {
+  fileMatch: ['(^|/)\\.terragrunt-version$'],
+  versioning: hashicorpVersioning.id,
+  extractVersion: '^v(?<version>.*)$',
```

Comment:
```suggestion
  extractVersion: '^v(?<version>.+)$',
```
version should at least have one char